src/domain/agent/verified-credential/dto/verified.credential.dto.result.ts

```diff
@@ -13,10 +13,10 @@ export class VerifiedCredential {
   })
   issuer: string;
 
-  @Field(() => Date, {
+  @Field(() => String, {
     description: 'The time at which the credential was issued',
   })
-  issued?: Date;
+  issued?: string;
```

Comment:
Why string and not date?

Reply:
Date was not resolving properly, giving that null error that we saw. It may be that the value needs to be converted in the wallet mgr before sending back, but in terms of getting it working this was the fastest route....